tests/stubtest_allowlists/py312.txt

```diff
@@ -280,39 +280,28 @@ _csv.Writer
 argparse._MutuallyExclusiveGroup.add_mutually_exclusive_group
 configparser.LegacyInterpolation.__init__
 enum.Enum.__init__
-# TODO: The stub for enum.auto is nothing like the implementation
-enum.auto.__init__
-enum.auto.value
-fractions.Fraction.__new__  # overload is too complicated for stubtest to resolve
 ftplib.FTP.trust_server_pasv_ipv4_address
-functools.cached_property.__set__  # Stub is a white lie; see comments in the stub
-functools._lru_cache_wrapper.cache_parameters  # Cannot be detected statically
-ipaddress.IPv4Interface.hostmask
-ipaddress.IPv6Interface.hostmask
 ipaddress._BaseNetwork.broadcast_address
 ipaddress._BaseNetwork.hostmask
+ipaddress.IPv4Interface.hostmask
+ipaddress.IPv6Interface.hostmask
 multiprocessing.managers.SharedMemoryServer.create
 multiprocessing.managers.SharedMemoryServer.list_segments
 multiprocessing.managers.SharedMemoryServer.public
 multiprocessing.managers.SharedMemoryServer.release_segment
 multiprocessing.managers.SharedMemoryServer.shutdown
 multiprocessing.managers.SharedMemoryServer.track_segment
 multiprocessing.spawn._main
-# platform.uname_result's processor field is now dynamically made to exist
 platform.uname_result.__match_args__
 platform.uname_result.__new__
-platform.uname_result.processor
+platform.uname_result.processor  # platform.uname_result's processor field is now dynamically made to exist
```

Comment:
This comment applies to the `platform.uname_result.__match_args__` and `platform.uname_result.__new__` allowlist entries as well (though I agree that isn't very clear at the moment)